massa-consensus/src/settings.rs

```diff
@@ -11,6 +11,9 @@ use serde::{Deserialize, Serialize};
 use std::str::FromStr;
 use std::{default::Default, path::PathBuf, usize};
 
+// Consensus static parameters( defined by protocol used)
+// Change one of the following values is a consider as a breaking change
```

Comment:
There is something off with that sentence maybe "Changing one of the following values is a considered as a breaking change"

Reply:
Right!